### ISSUE TITLE
demo with grpc

### ISSUE THREAD

**Issue by expe-elenigen** [phiSgr/gatling-ext#4]
@phiSgr Hi, we were using your other lib for grpc and I noticed you've decided to deprecate it since Gatling supports now grpc natively with the Enterrpise license, the problem I have is we don't have an Enterprise version which means our test could only run for 5min with 5 users with their library.  I feel that it would still make sense to keep your [phiSgr/gatling-grpc](https://github.com/phiSgr/gatling-grpc) active so there would still have at least one free solution on the market. At the same time, I see that you have created a flexible solution in this new repository, would it be possible to have a demo project on how to use it for example with grpc?
Thank you!

**Comment by phiSgr** [comment]
I noted the reasons for deprecating the project in the following comments.

https://github.com/phiSgr/gatling-grpc/issues/88#issuecomment-1931266825

https://community.gatling.io/t/bug-report-for-the-grpc-plugin/8690/17

Finally:


https://github.com/phiSgr/gatling-grpc/blob/4d24031c08d9dc73f0b224ac3856037c92d0b860/README.md?plain=1#L8

**Comment by expe-elenigen** [comment]
Thank you for taking the time to clarify this point, I totally missed your intention here. I feel the message you left in the repository sounds like the project is no longer needed and can be replaced seamlessly with the official one, which isn't really the case for those using the community edition setup. While I understand your decision to avoid competing with their enterprise solution, I think it would be appreciated to highlight that for people who want to migrate that they would need the enterprise license, or they could stick for now on this project if they are fine with the fact it's archived.

If I understand you correctly, you don't plan to close this project even if it's partially competing with their gRPC solution, since it's not targeting the exact same goal? I'm thinking to convert our project to give a try to this other solution, but if you are considering to close it, I guess we would just have to find a new solution.

**Comment by phiSgr** [comment]
This library (gatling-ext) allows you to use Gatling to measure the running time of any (blocking/non-blocking) function on the JVM. Yes, those functions can be gRPC calls; but there's no integration with gRPC (e.g. status codes and headers).\
So I don't really think it is competing with Gatling Corp's gRPC implementation.

I'm not going to shut this project down, but I don't think I will update it any time soon.

---

> I think it would be appreciated to highlight that for people who want to migrate that they would need the enterprise license

The documentation I linked to has "Enterprise" next to "gRPC". I did not think I have to note that in my README. Especially my README won't be changed any more, but Gatling's licensing may change (not that it's likely).

Also I said "Feel free to fork and maintain it if you want (to provide) up-to-date free stuff". One can infer the alternative is probably not free (in both senses of the word).
